staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -320,6 +320,10 @@ func TestErrConnKilled(t *testing.T) {
 	if !strings.Contains(capturedOutput, `timeout or abort while handling: method=GET URI="/" audit-ID=""`) {
 		t.Errorf("unexpected output captured actual = %v", capturedOutput)
 	}
+	t.Cleanup(func() {
```

Comment:
I think this is incorrect. If we fail on `http.Get` and `t.Fatal` will be invoked this `t.Cleanup` will never be executed and you won't close the server and execute `cleanUp` function. I think you should place the `t.Cleanup` in the same place there was a `defer ts.Close()`.

Reply:
Thanks, your comment is correct, so I fixed it.